### ISSUE TITLE
Tslint is enabled in files with mjs extension

### ISSUE THREAD

**Issue by Yanpas** [Microsoft/vscode-tslint#418]
- tslint Version: 1.0.42
- typescript Version: 3.2

For some reason tslint is enabled if I open fil with name ending with .mjs despite mjs is javascript code.

**Comment by timc1** [comment]
Hey Yan, have you solved this issue yet? 🧐I'm running into a similar issue, thanks! :)

**Comment by Yanpas** [comment]
No, didn't find a solution, but haven't tried newer vscode and extension

**Comment by timc1** [comment]
Ahh, ok! I'll update this when I find a solution on my end!

**Comment by egamma** [comment]
@Yanpas this issue is filed against the wrong extension. I'll move it.
